packages/cloud_functions/cloud_functions/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.4.2
```

Comment:
nit: since this doesn't introduce any new Dart API surface area I think it should be 0.4.1+7.

Reply:
Okay, that's an easy change to make. I'll upload it shortly.